### PR TITLE
osmcoastline 2.3.0 (new formula)

### PR DESCRIPTION
**Pull request by kkdd**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by kkdd — COMMENTED** [review]
license "GPL-3.0"

**Review by kkdd — COMMENTED** [review]
"GPL-3.0-only"

File: Formula/osmcoastline.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Osmcoastline < Formula
+  desc "Extracts coastline data from OpenStreetMap planet file"
+  homepage "https://osmcode.org/osmcoastline/"
+  url "https://github.com/osmcode/osmcoastline/archive/v2.3.0.tar.gz"
+  sha256 "b08ddcc8cb494a19cbf4d9f1501c47443fe374a4fe171e6f55c730ca1e710689"
+  license "GPL-3.0-only"
```
Comment (carlocab) [inline Formula/osmcoastline.rb:6]:
```suggestion
  license "GPL-3.0-or-later"
```
Seems like this is `-or-later`. See https://github.com/osmcode/osmcoastline/search?q=license

**Review by kkdd — COMMENTED** [review]
license "GPL-3.0-or-later"

File: Formula/osmcoastline.rb (hunk 1 above)
Comment (kkdd) [reply Formula/osmcoastline.rb]: Thank you for you suggestion.
+    license "GPL-3.0-or-later"

**Review by carlocab — DISMISSED** [review]
Thanks, @kkdd.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**